packages/features/webhooks/pages/webhook-edit-view.tsx

```
@@ -95,7 +95,8 @@ export function EditWebhookView({ webhook }: { webhook?: WebhookProps }) {
             timeUnit: values.timeUnit,
           });
         }}
-        apps={installedApps?.items.map((app) => app.slug)}
+        apps={[...installedApps?.items.map((app) => app.slug), "cal-video"]}
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)